[2025-01-26T09:17:27.536+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T09:17:27.650+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_dag_test.branch manual__2025-01-26T09:17:24.569715+00:00 [queued]>
[2025-01-26T09:17:27.655+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_dag_test.branch manual__2025-01-26T09:17:24.569715+00:00 [queued]>
[2025-01-26T09:17:27.657+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-26T09:17:27.666+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): branch> on 2025-01-26 09:17:24.569715+00:00
[2025-01-26T09:17:27.671+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=784) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-26T09:17:27.672+0000] {standard_task_runner.py:72} INFO - Started process 786 to run task
[2025-01-26T09:17:27.672+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'final_dag_test', 'branch', 'manual__2025-01-26T09:17:24.569715+00:00', '--job-id', '439', '--raw', '--subdir', 'DAGS_FOLDER/t.py', '--cfg-path', '/tmp/tmpzn_iyqa6']
[2025-01-26T09:17:27.674+0000] {standard_task_runner.py:105} INFO - Job 439: Subtask branch
[2025-01-26T09:17:27.704+0000] {task_command.py:467} INFO - Running <TaskInstance: final_dag_test.branch manual__2025-01-26T09:17:24.569715+00:00 [running]> on host 7854821b9569
[2025-01-26T09:17:27.755+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='final_dag_test' AIRFLOW_CTX_TASK_ID='branch' AIRFLOW_CTX_EXECUTION_DATE='2025-01-26T09:17:24.569715+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-26T09:17:24.569715+00:00'
[2025-01-26T09:17:27.757+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T09:17:27.765+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-26T09:17:27.771+0000] {python.py:240} INFO - Done. Returned value was: filtering.filter_by_southeast
[2025-01-26T09:17:27.772+0000] {branch.py:38} INFO - Branch into filtering.filter_by_southeast
[2025-01-26T09:17:27.780+0000] {skipmixin.py:230} INFO - Following branch ('filtering.filter_by_southeast',)
[2025-01-26T09:17:27.785+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-01-26T09:17:27.804+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T09:17:27.805+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=final_dag_test, task_id=branch, run_id=manual__2025-01-26T09:17:24.569715+00:00, execution_date=20250126T091724, start_date=20250126T091727, end_date=20250126T091727
[2025-01-26T09:17:27.846+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-26T09:17:27.861+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-26T09:17:27.864+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
